base_station/gui/src/components/AutonTask.vue

```diff
@@ -24,8 +24,10 @@
      <h1>Nav State: {{this.nav_status.nav_state_name}}</h1>
      <div class="raw-data raw-sensors">
         <RawSensorData v-bind:GPS="GPS" v-bind:IMU="IMU"/>
-        <RadioSignalStrength v-bind:RadioSignalStrength="RadioSignalStrength"/>
+        <br>
+        <br>
         <Obstacle v-bind:Obstacle="Obstacle"/>
+        <br>
```

Comment:
@jnnanni I had to add these breaklines to avoid overlapping text for some reason. Not really sure what happened here, at some point this week can you look at it? We shouldn't need to use breaklines to do this. (Not necessary to change before merging this PR I don't think)